comscore/src/integration/ComscoreStreamingAnalyticsTHEOIntegration.ts

```diff
@@ -512,8 +549,15 @@ export class ComscoreStreamingAnalyticsTHEOIntegration {
         }
     };
 
-    private isBeforePreRoll = (): boolean =>
-        this.player.ads?.scheduledAdBreaks.length ? this.player.ads?.scheduledAdBreaks[0].timeOffset === 0 : false;
+    private isBeforePreRoll = (): boolean => {
+        const hasScheduledAdBreaks = this.player.ads?.scheduledAdBreaks.length !== 0;
```

Comment:
Can `player.ads` ever be undefined here? Might want to add an early return if that's the case?

Reply:
✅